build.gradle

```diff
@@ -240,6 +240,9 @@ subprojects {
                 allowInsecureProtocol = true
                 url "${localNexusLocation}/public"
             }
+            maven {
+                url "https://oss.sonatype.org/content/groups/staging/"
```

Comment:
Do we need this in the master branch?

Reply:
IMHO its the same reasoning as for private nexus... sonatype repo gives me a chance to test public artifact before I click on release